### DISCUSSION TITLE
Make flags appear in nicknames

### DISCUSSION THREAD

**Discussion by Thopla99** [Chatterino/chatterino2#3248] [Ideas] [answered]
Is it possible to make so flag emojis appear in nicknames like normal emojis do ?

**Comment by Mm2PL** [comment, marked as answer]
Emoji actually aren't processed in nicknames. If you have the required fonts, yes they will show but it's not the same as emoji in chat.

**Reply by Thopla99** [reply]
Ohhhh makes sense thats why emojis dont appear the same way they do in chat thanks for enlightening me, i feel so dumb now thanks.
